doc/source/myrocks/variables.rst

```diff
@@ -628,6 +663,18 @@ until there is less than 10 bits of free space remaining.
 
 Allowed range is from ``1`` to ``2147483647``.
 
+.. variable:: rocksdb_bulk_load_allow_unsorted
+
+  :version 5.7.20-18: Implemented
+  :cli: ``--rocksdb-bulk-load-allow-unsorted``
+  :dyn: Yes
+  :scope: Global, Session
+  :vartype: Boolean
+  :default: ``OFF``
+
+When enabled, unsorted inputs are allowed, which are then sorted by
```

Comment:
By default, the bulk loader requires its input to be sorted in the primary key order. If enabled, unsorted inputs are allowed too, which are ...